### PR TITLE
Import Firefox 56 schemas

### LINKED ISSUE #1346: Update schemas to Firefox 56
Once Firefox 56 is on Beta we should import the schemas, cut a new release and update web-ext to use it for linting.

If there are interesting things landing in Firefox 56 please add a comment so it's a bit easier to track what's incoming.

### PR DESCRIPTION
**Pull request by mstriemer**
Import Firefox 56 schemas again, hopefully with a review this time.

This reverts commit 9f49c76a5ca130f3f760df1113c5d3b264d0fe34.
Fixes #1346.

### CODE REVIEW

File: src/schema/imported/manifest.json [hunk 1]
```diff
@@ -162,13 +162,17 @@
...
-                    "type": "string",
-                    "deprecated": "Unknown permission ${value}",
+                    "onError": "warn"
+                  }
+                ],
+                "anyOf": [
+                  {},
+                  {
                     "format": "deprecated"
                   }
```
Comment (EnTeQuAk) [inline src/schema/imported/manifest.json]: this `anyOf` block looks weird? Or is both together essentially warning with a deprecation error or something?

File: src/schema/imported/tabs.json [hunk 2]
```diff
@@ -361,7 +362,7 @@
             },
             "highlighted": {
               "type": "boolean",
-              "description": "Whether the tabs are highlighted."
+              "description": "Whether the tabs are highlighted.  Works as an alias of active."
```
Comment (EnTeQuAk) [inline src/schema/imported/tabs.json]: doh! double whitespaces! I know it's a import but things like that jump directly into my eyes

File: src/schema/imported/manifest.json (hunk 1 above)
Comment (mstriemer) [inline src/schema/imported/manifest.json]: It is definitely weird, I'm not sure what's causing it but it looks like the `allOf` is handled first and the permissions tests (that aren't broken anyway) pass.

File: src/schema/imported/tabs.json (hunk 2 above)
Comment (mstriemer) [inline src/schema/imported/tabs.json]: Good catch, this will have to be changed in Firefox though.

Comment (EnTeQuAk) [inline src/schema/imported/tabs.json]: yeah, I'm not too worried about that tbh :)

**Comment by EnTeQuAk** [comment]
@mstriemer lgtm of this?

**Comment by EnTeQuAk** [comment]
I'm going to merge this and issue a new release to get this into todays tag and resolve #1486

**Pull request merged**